src/interface/ui/base/List.hpp

```diff
@@ -226,7 +225,14 @@ class List : virtual public ExternalInterface,
 
         return std::future_status::ready == startup_future_.wait_for(none);
     }
-    auto wait_for_startup() const noexcept -> void { startup_future_.get(); }
+    auto wait_for_startup() const noexcept -> void
+    {
+        try {
+            startup_future_.get();
+        } catch (const std::exception&) {
+            // TODO MT-34 improve
```

Comment:
I suggest to print what went wrong before MT-34 will be implemented

Reply:
More is needed, hence the TODO and ticket, but I have added a minimal diagnostic line especially for you :-)